Birdui/Birdui/PostListView.swift

```diff
@@ -9,10 +9,56 @@
 import SwiftUI
 
 struct PostListView: View {
+    @State var modalIsPresented = false
+
+    struct HeaderTitleTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .foregroundColor(Color.black)
+                .font(.system(size: 17))
+        }
+    }
+
+    struct ButtonTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .font(.system(size: 15))
+        }
+    }
+
     var body: some View {
-        // TODO: This should look exactly like the Birdie table view,
-        // but with only one button.
-        Text("Layout header, new-post button, List of posts")
+        NavigationView {
+        VStack(alignment: .leading) {
+            let leftImageSize = 50
+            HStack {
+                Image("mascot_swift-badge")
+                    .resizable()
+                    .frame(width: 50, height: 50)
+                Spacer()
+                Text("Home")
+                    .modifier(HeaderTitleTextStyle())
+                    .padding(.leading, -CGFloat(leftImageSize))
+                Spacer()
+            }
+            HStack {
+                Button(action: { self.modalIsPresented = true }) {
+                    Text("Create New post")
+                        .modifier(ButtonTextStyle())
+                }
+            }
+            List {
+                Text("Layout header, new-post button, List of posts")
+            }
+            .navigationBarTitle("")
+            .navigationBarHidden(true)
+        }
+        .padding(.top, 16)
+        .padding(.leading, 24)
+        .padding(.trailing, 24)
```

Comment:
Because `List` has built-in `contentInsets`, adding extra padding makes too much white-space around it. Maybe it would be better if we applied the padding only to the header part?